docs/contributing/coding-standards/component-options.md

```diff
@@ -231,37 +69,12 @@ In our example, `rememberExpanded` becomes `data-remember-expanded`.
 
 ### In Nunjucks
 
-Unlike the `data-*` attribute in HTML and our use of `dataset` in JavaScript, there is no intrinsic link between the Nunjucks parameter name and the names used elsewhere. The Nunjucks parameter can therefore be named differently, if convenient.
+Unlike the `data-*` attribute in HTML, there is no intrinsic link between the Nunjucks parameter name and the names used elsewhere. The Nunjucks parameter can therefore be named differently, if convenient.
```

Comment:
**suggestion** Dropping 'and our use of `dataset`...' sounds good, but I think it needs replacing with something to explain we're talking about the link between the name of data attributes and JavaScript option names.
```suggestion
Unlike the `data-*` attribute in HTML and the name of JavaScript configuration options, there is no intrinsic link between the Nunjucks parameter name and the names used elsewhere. The Nunjucks parameter can therefore be named differently, if convenient.
```